src/containers/forms/OrganizationSelect/tests/__snapshots__/index.test.tsx.snap

```diff
@@ -0,0 +1,1391 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
This snapshot file is much too large - can we reduce it?  I think the sweet spot in terms of size is 100 or so lines max per snapshot test in this file.  There's a lot of clutter from Formik that needs to be removed, for instance.

Reply:
@moshthepitt Yeah i would really like to, it's not clear to me what identifier I can use to try and get snippets of the whole thing.

Reply:
Replaced with smaller snapshots of the props instead of the component itself (and all its children).